nuxeo-runtime/nuxeo-runtime-stream/src/test/java/org/nuxeo/runtime/stream/tests/TestAvroSchemaStoreService.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * (C) Copyright 2018 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     pierre
+ */
+package org.nuxeo.runtime.stream.tests;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+import javax.inject.Inject;
+
+import org.apache.avro.Schema;
+import org.apache.avro.SchemaNormalization;
+import org.apache.avro.message.SchemaStore;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.nuxeo.runtime.api.Framework;
+import org.nuxeo.runtime.avro.AvroSchemaStoreService;
+import org.nuxeo.runtime.test.runner.Deploy;
+import org.nuxeo.runtime.test.runner.Features;
+import org.nuxeo.runtime.test.runner.FeaturesRunner;
+import org.nuxeo.runtime.test.runner.RuntimeFeature;
+
+/**
+ * @since 10.2
+ */
+@RunWith(FeaturesRunner.class)
+@Features(RuntimeFeature.class)
+@Deploy("org.nuxeo.runtime.stream")
+@Deploy("org.nuxeo.runtime.stream:test-avro-contrib.xml")
+public class TestAvroSchemaStoreService {
+
+    @Inject
+    public AvroSchemaStoreService service;
+
+    @Test
+    public void testSchemaStoreRetrieval() {
+        Assert.assertNotNull(Framework.getService(SchemaStore.class));
```

Comment:
Would it also work for `org.nuxeo.runtime.avro.AvroSchemaStoreService` ?

Reply:
Yes it is what's used by @Inject but it may be clearer to assert both, will do

Reply:
Afaik `@Inject` will check if bean exists and fail otherwise.

Reply:
Yes you'll get a test setup failure if the injection fails to return a non-`null` value, so the assertion is not needed of you already inject the same service.